src/main/java/brave/opentracing/BraveTracer.java

```diff
@@ -76,6 +94,11 @@ public static Builder newBuilder(Tracing brave4) {
       formatToPropagation.put(Format.Builtin.TEXT_MAP, brave4.propagation());
     }
 
+    public Builder setActiveSpanSource(ActiveSpanSource activeSpanSource) {
```

Comment:
we don't use set prefix on things that are not javabeans (which means basically all code)

PS add javadoc on new public symbols at least linking to where to find more.